src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/ConfigureEndpointAzureStorageQueueTransport.cs

```diff
@@ -13,7 +13,7 @@ public class ConfigureEndpointAzureStorageQueueTransport : IConfigureEndpointTes
     {
         var connectionString = settings.Get<string>("Transport.ConnectionString");
         //connectionString = "UseDevelopmentStorage=true;";
-        configuration.UseSerialization<XmlSerializer>();
+        configuration.UseSerialization<JsonSerializer>();
```

Comment:
> Reminding that Xml is highly unlikely to be used in Azure scenarios.

as long as it's not unsupported, I don't think this statement should have any influence on this discussion.

so this is clearly not a fix but a workaround and I don't see any bugs raised/updated? I'm good with moving forward with this PR but then it seems we need to raise an issue about XmlSerializer still broken or reopen. I wasn't involved in the investigation so maybe @dbelcham and @SeanFeldman can chime in?